app/controllers/works_controller.rb

```diff
@@ -148,7 +148,7 @@ def create_new_version(previous_version)
     previous_version.dup.tap do |work_version|
       work_version.state = 'version_draft'
       work_version.version = previous_version.version + 1
-      CollectionObserver.collection_activity(work_version, nil)
+      CollectionObserver.new_version_draft_created(work_version, nil)
```

Comment:
would "version_draft_created" be sufficient, as "created" implies "new"?   "updated" would imply changes ...